File: String/codeforces_110A.cpp

// Problem link ===>> https://codeforces.com/problemset/problem/110/A
// submission link ===>> https://codeforces.com/contest/110/submission/224934565

#include <iostream>
#include <string>
using namespace std;

int main()
{
    string n;
    cin >> n;

    int count = 0;
    for (int i = 0; i < n.length(); i++)
    {
        if (n[i] == '4' || n[i] == '7')
        {
            count++;
        }
    }

    if (count == 4 || count == 7)
    {
        cout << "YES" << endl;
    }
    else
    {
        cout << "NO" << endl;
    }
    return 0;
}

File: String/codeforces_110A.py

# Problem link ===>> https://codeforces.com/problemset/problem/110/A

def is_nearly_lucky(n):
  """Returns True if the input number is nearly lucky, False otherwise."""
  count = 0
  for digit in n:
    if digit == '4' or digit == '7':
      count += 1
  return count == 4 or count == 7

n = input()
if is_nearly_lucky(n):
  print('YES')
else:
  print('NO')
